resource-server/src/main/java/com/okta/spring/example/ResourceServerExampleApplication.java

```diff
@@ -0,0 +1,63 @@
+package com.okta.spring.example;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.oauth2.resource.ResourceServerProperties;
```

Comment:
nit: unused imports in the file - 

ResourceServerProperties
ResourceServerSecurityConfigurer
RequestMethod